packages/client/src/user/userReducer.ts

```diff
@@ -65,21 +65,15 @@ const initialState: IUserFormState = {
   userDetailsStored: false,
   submitting: false,
   loadingRoles: false,
+  userRoles: [],
```

Comment:
This could also be a React hook.. We just need a good template example of how to write offline-capable data fetch hooks 🤔

Reply:
Why do we need to know the user roles? Can't we get the scopes from the JWT token?